html/controls/pt/index.js

```diff
@@ -309,12 +308,13 @@
 	}
 
 	function submitPenalty() {
-		var p = $('.PenaltyEditor .Period').val();
-		var j = $('.PenaltyEditor .Jam').val();
-		var c = $('.PenaltyEditor .Codes .Active').attr('code');
-		var fo_exp = penaltyType === 'FO_EXP';
-
-		WS.Command("Penalty", { teamId: teamId, skaterId: skaterId, penaltyId: penaltyId, period: p, jam: j, code: c, fo_exp: fo_exp });
+		var sortKey = 'ScoreBoard.Team(' + teamId + ').Skater(' + skaterId + ').SortPenalties';
```

Comment:
This feels messy, and also could be racey if there's two PTs. I think we need a way to take this all in at once